specification/servicenetworking/ServiceNetworking.Management/examples/2024-05-01-preview/SecurityPolicyDelete.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Delete SecurityPolicy",
+  "operationId": "SecurityPoliciesInterface_Delete",
+  "parameters": {
+    "api-version": "2024-05-01-preview",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "trafficControllerName": "tc1",
+    "securityPolicyName": "sp1"
+  },
+  "responses": {
+    "200": {},
+    "202": {
+      "headers": {
+        "Retry-After": 3,
+        "location": "https://blobLocation.com"
```

Comment:
nitpick about your docs, this seems like probably an unrealistic example that you'd redirect to here